R/get_negative_cases.R

```diff
@@ -0,0 +1,22 @@
+#' Find all observations with negative incidence in a
+#' given data frame
+#'
+#' @param data data frame
+#' @return a data frame of cases with locations and dates
+#'
+#' @export
+#'
+get_negative_cases <- function(data) {
+  locations <- c()
+  dates <- c()
+
+  # Find  observations with negative inc
+  for (i in 1:nrow(data)) {
+    if (data[i, ]$inc < 0) {
+      locations <- c(locations, as.character(data[i, ]$location))
+      dates <- c(dates, as.character(data[i, ]$date))
+    }
+  }
```

Comment:
This could probably be done more easily (and more readable) via calls to `dplyr::filter` and `dplyr::transmute`.  For example, something like

```
return(
  data %>%
    dplyr::filter(inc < 0) %>%
    dplyr::transmute(
      fips = location,
      dates = date
    )
)
```

But also note that for readability it might be nice to keep consistent names (i.e., `location` and `date` rather than `fips` and `dates`), unless there are specific reasons it's advantageous to change the names.